pkg/exposedsecretreport/builder.go

```diff
@@ -0,0 +1,112 @@
+package exposedsecretreport
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/aquasecurity/trivy-operator/pkg/apis/aquasecurity/v1alpha1"
+	"github.com/aquasecurity/trivy-operator/pkg/kube"
+	"github.com/aquasecurity/trivy-operator/pkg/trivyoperator"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/validation"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+type ReportBuilder struct {
+	scheme     *runtime.Scheme
+	controller client.Object
+	container  string
+	hash       string
+	data       v1alpha1.ExposedSecretReportData
+	reportTTL  *time.Duration
+}
+
+func NewReportBuilder(scheme *runtime.Scheme) *ReportBuilder {
+	return &ReportBuilder{
+		scheme: scheme,
+	}
+}
+
+func (b *ReportBuilder) Controller(controller client.Object) *ReportBuilder {
+	b.controller = controller
+	return b
+}
+
+func (b *ReportBuilder) Container(name string) *ReportBuilder {
+	b.container = name
+	return b
+}
+
+func (b *ReportBuilder) PodSpecHash(hash string) *ReportBuilder {
+	b.hash = hash
+	return b
+}
+
+func (b *ReportBuilder) Data(data v1alpha1.ExposedSecretReportData) *ReportBuilder {
+	b.data = data
+	return b
+}
+
+func (b *ReportBuilder) ReportTTL(ttl *time.Duration) *ReportBuilder {
```

Comment:
Not sure if severity `unknown` is relevant to secret world. its more related to vulnerability scanning when it comes from an external DB and its, in secret we define the severity so unknown not likely to happen